content/cnc-router.md

```diff
@@ -0,0 +1,50 @@
+# The CNC Router - [X-Carve](https://shop.inventables.com/products/x-carve-1)
+
+The X-Carve CNC router works on an XY plane with limited Z-depth, capable of handling dimensions of 1000mm x 1000mm x 50mm. It is specifically designed to cut wood, and in part due to our dust collection system, should not be used for other materials.  
+Contrary to a typical CNC machine, our CNC router is designed to move relatively slowly. It relies on tool paths for rounded edges or complex shapes.
+
+## ⚠️ Safety
+
+1. Always keep the router's door closed when power is supplied to it.
+2. Secure your workpiece; if it isn't secure, it could go flying. Use a fixture or at least two clamps on each side of your piece.
```

Comment:
Is the tape worth a mention since it will probably be used?

Reply:
Covered in the docs from my NYIC PR